bgt/game_mechanics.py

```diff
@@ -79,6 +79,8 @@ class Feedback(BaseModel):
     is_correct: bool = None
     response_text: str = None
 
+    # question: does it make sense for the following to be @validators?
+    # maybe they should be @properties
```

Comment:
Based on a quick look at the pydantic documentation, it looks like @pydantic.property would make sense.